packages/core/src/Layout/Layout.ts

```diff
@@ -41,21 +54,49 @@ class Layout {
       };
       pageSources = new PageSources();
       this.layoutPageBodyVariable = `{{${uuidv4()}-${uuidv4()}}}`;
-      const layoutVariables = { [LAYOUT_PAGE_BODY_VARIABLE]: this.layoutPageBodyVariable };
+      const layoutVariables = {
+        [LAYOUT_PAGE_BODY_VARIABLE]: this.layoutPageBodyVariable,
+      };
 
-      const { variableProcessor, pluginManager, siteLinkManager } = this.config;
+      const {
+        variableProcessor,
+        pluginManager,
+        siteLinkManager,
+      }: {
+        variableProcessor: VariableProcessor;
+        pluginManager: any;
+        siteLinkManager: any;
+      } = this.config;
 
       const nunjucksProcessed = variableProcessor.renderWithSiteVariables(
-        this.sourceFilePath, pageSources, layoutVariables);
-      nodeProcessor = new NodeProcessor(fileConfig, pageSources, variableProcessor,
-                                        pluginManager, siteLinkManager, this.layoutUserScriptsAndStyles,
-                                        'layout');
+        this.sourceFilePath,
+        pageSources,
+        layoutVariables,
+      );
+      nodeProcessor = new NodeProcessor(
+        fileConfig,
```

Comment:
Minor nit, why are all the params on a line on its own? 😅 Is there a reason for making this change throughout the file? ESLint doesn't seem to mind the original way

Reply:
i think my vs code was giving me some linting tips haha, so I went and made the change

Reply:
I think it would be good to revert some of these formatting changes for better code readability.